Runtime/Model/BacktraceHttpClient.cs

```diff
@@ -167,7 +167,14 @@ private void AddAttachmentToFormData(List<IMultipartFormSection> formData, IEnum
 
             foreach (var file in uniqueAttachments)
             {
-                if (string.IsNullOrEmpty(file) || File.Exists(file) == false || new FileInfo(file).Length > maximumAttachmentSize)
+                if (string.IsNullOrEmpty(file) || File.Exists(file) == false)
+                {
+                    continue;
+                }
+
+                var fileSize = new FileInfo(file).Length;
+                var isInvalidFileSize = fileSize > maximumAttachmentSize || fileSize == 0;

```

Comment:
Maybe it's worth separately logging if `fileSize > maximumAttachmentSize`?

Reply:
We didn't change this logic in this pull request ( we just added a condition to check if fileSize is equal to zero. Not sure if this is a good idea because we can spam too many logs when the game will generate unhandled exceptions in the update loop.

Reply:
Ah okay, I just brought it up because it may confuse the user when they are missing some file attachment, but I guess this condition `fileSize > maximumAttachmentSize` occurs mostly when there are unhandled exceptions in update loop so it won't happen often.